pubs_of_sheffield.php

```diff
@@ -2,17 +2,16 @@
 require 'functions.php';
 $db = connectToDatabase();
 $itemsList = getAllItems($db);
+session_start()
```

Comment:
You can axe this, session isn't needed to use forms, only when you want to start tracking info about a user like whether or not they are logged in